Main.js

```diff
@@ -591,14 +591,27 @@ function open_player_sheet(sheet_url) {
 				conditions.push($(this).text());
 			});
 
+			abilities = [];
+			$(event.target).contents().find('.ct-quick-info__ability').each(function() {
+				abilities.push({
+					abilityName: $(this).find('.ddbc-ability-summary__label').text(),
+					abilityAbbr: $(this).find('.ddbc-ability-summary__abbr').text(),
+					modifier: `${$(this).find('.ddbc-signed-number__sign').text()}${$(this).find('.ddbc-signed-number__number').text()}`,
+					score: $(this).find('.ddbc-ability-summary__secondary').text()
+				});
+			});
+
 
 			var playerdata = {
 				id: sheet_url,
 				hp: current_hp,
 				max_hp: max_hp,
 				ac: $(event.target).contents().find(".ddbc-armor-class-box__value").html(),
 				pp: pp.html(),
-				conditions: conditions
+				conditions: conditions,

```

Comment:
NIT
Can change this to just `conditions` as you did with `abilities` below to keep it consistent.

Reply:
yep you're right, ahahah but I didn't read it only added my properties.